cmd/thv/app/rm.go

```diff
@@ -37,9 +38,37 @@ func rmCmdFunc(cmd *cobra.Command, args []string) error {
 	}
 
 	// Delete container.
-	if err := manager.DeleteContainer(ctx, containerName, rmForce); err != nil {
+	if err := manager.DeleteContainer(ctx, containerName, rmForce, true); err != nil {
 		return fmt.Errorf("failed to delete container: %v", err)
 	}
 
+	// Delete associated egress container.
+	egressContainerName := containerName + "-egress"
+	if err := manager.DeleteContainer(ctx, egressContainerName, rmForce, false); err != nil {
```

Comment:
Should this logic go inside "DeleteContainer"? It's an implementation detail of the docker driver after all.